test/functional/mempool_datacarrier.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020-2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test datacarrier functionality"""
+from test_framework.messages import (
+    CTxOut,
+    CTransaction,
```

Comment:
yocto-nit: sorted order
```suggestion
    CTransaction,
    CTxOut,
```